src/rules/noMagicNumbersRule.ts

```diff
@@ -91,4 +97,9 @@ class NoMagicNumbersWalker extends Lint.AbstractWalker<Set<string>> {
             this.addFailureAtNode(node, Rule.FAILURE_STRING);
         }
     }
+
+    private isNumberPrototypeMethod(methodName: string): boolean {
+        const numberMethods = ["toExponential", "toFixed", "toPrecision", "toString"];
```

Comment:
make this a static const by defining it outside the class, instead of redefining every time this function is called. then you actually don't even need this method.

```ts
const NUMBER_METHODS = [...];
class {
    ... && NUMBER_METHODS.indexOf(methodName) >= 0
}
```